docs/topics_5/web-analyze-apps.adoc

```diff
@@ -3,17 +3,11 @@
 [id='using_web_console_analyze_apps_{context}']
 = Analyzing applications with the {WebName}
 
-In order to use the {WebName} to analyze applications, you must create a project. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
+To use the {WebName} to analyze applications, you must create one or more projects. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
 
-Follow these steps to use the {WebName} to analyze your applications and review the results.
+Follow these steps to use the {WebName} to create projects, analyze your applications and review the results:
 
-. Add a project.
-. Add applications to the project.
-. Specify the analysis configuration.
-. Execute the analysis.
+. Create a project.
+. Run the analysis.
 . View the results of the analysis.
-
-[NOTE]
-====
-Click the *Projects* link at the top of the {WebName} to return to the project list at any time. Visit the help menu for an *About* page and a link to the {ProductShortName} documentation.
-====
+. If needed, reconfigure the analysis and run the analysis again.
```

Comment:
That's not a step. That's an option

Reply:
I think the ability to reconfigure the analysis without creating a new project is a good reason. If you disagree, I will remove it.

Reply:
I would add it to the previous step:
+
You can reconfigure the analysis and run it again.

It doesn't merit its own step